build/android/proj/libadmob/src/com/cocos/admob/proto/openapp/AppOpenAdFullScreenContentCallbackNTF.java

```diff
@@ -0,0 +1,16 @@
+package com.cocos.admob.proto.openapp;
```

Comment:
Can we rename the package name to `appopen` as well? (the same applies to all files in the same directory)